rundeckapp/grails-app/services/rundeck/services/LogFileStorageService.groovy

```diff
@@ -2243,6 +2245,25 @@ class LogFileStorageService
     boolean forcePartialCheckConfig() {
         configurationService.getBoolean(FORCE_PARTIAL_CHECKING, false)
     }
+
+    private Set<Long> getExecIds (String serverUUID, Set<Long> skipExecIds) {
```

Comment:
What is the reason for this method? It seems redundant for the functionality already provided in the log file storage provider.
Also, it adds a direct call to a Gorm object which we'd like to avoid doing if possible.